scripts/web_component_studio/web_component_studio.py

```diff
@@ -51,6 +52,23 @@ def run_webots():
     command = 'webots' if os.name == 'nt' else WEBOTS_HOME + '/webots'
     os.system(command + ' --enable-x3d-meta-file-export --mode=fast --no-rendering --minimize ' + WORLD)
 
+def convert_local_resources(remote_address, type):
+    """Convert local textures or meshes to remote ones"""
+    project_path = address.split("://")[1]
+    resources_path = os.path.join(WEBOTS_HOME, project_path, type + '/')
+    resources_path2 = os.path.join(WEBOTS_HOME, project_path, component['proto'], type +'/')
+    if os.path.isdir(resources_path):
+        search_and_replace(os.path.join(WEBOTS_HOME, 'docs', 'guide', 'scenes', component['name'],
+                                        component['name'] + '.x3d'), '"' + type + '/',
+                           '"' + remote_address + '/' + type + '/')
+    elif os.path.isdir(resources_path2):
+        search_and_replace(os.path.join(WEBOTS_HOME, 'docs', 'guide', 'scenes', component['name'],
+                                        component['name'] + '.x3d'), '"' + type + '/',
+                           '"' + remote_address + '/' + component['proto'] + '/' + type + '/')
+
+    resources_path = os.path.join(WEBOTS_HOME, 'docs', 'guide', 'scenes', component['name'], type)
+    if os.path.isdir(resources_path):
+        shutil.rmtree(resources_path)
```

Comment:
Don't we need to remove also this folder if it exists?
```suggestion
    resources_path = os.path.join(WEBOTS_HOME, 'docs', 'guide', 'scenes', type)
    if os.path.isdir(resources_path):
        shutil.rmtree(resources_path)
```

Reply:
This folder does not exist because the textures/meshes folder is always at the same level that the `.x3d` file as it is created like a scene/animation.